docs/build/advanced-concepts/metadata-views.md

```diff
@@ -0,0 +1,226 @@
+```
+
+### `resolveView` Function
+
+Regardless of whether it's used at the contract or NFT level, this function's role is to deliver the actual metadata associated with a given view type.
+
+```cadence
+pub fun resolveView(_ view: Type): AnyStruct? {
+    switch view {
+        case Type<MetadataViews.Display>():
+            ...
+        ...
+    }
+    return nil
+}
+```
+
+## NFT-Level Metadata Implementation
+
+NFT-level metadata addresses the unique attributes of individual tokens within a collection. It provides structured information for each NFT, including its identifier, descriptive elements, royalties, and other associated metadata. Implementing this level of detail ensures consistency and standardization across individual NFTs, making them interoperable and recognizable across various platforms and marketplaces.
+
+### Core Properties
+
+Every NFT has core properties that define its basic attributes. For example, in the provided code, an NFT has properties like its unique ID, name, description, and more. When we add the `MetadataViews.Resolver` to our NFT resource, we are indicating that these variables will align with the specifications defined in the `MetadataViews` contract for each of these properties. This ensures interoperability within the Flow ecosystem and guarantees that the metadata of our NFT can be consistently accessed and understood by various platforms and services that interact with NFTs.
+
+```cadence
+pub resource NFT: NonFungibleToken.INFT, MetadataViews.Resolver {
+    pub let id: UInt64
+    pub let name: String
+    pub let description: String
+    pub let thumbnail: String
+    access(self) let royalties: [MetadataViews.Royalty]
+    access(self) let metadata: {String: AnyStruct}
+    ...
+}
+```
+
+### Metadata Views for NFTs
+
+`MetadataViews` types define how the NFT presents its data. When called upon, the system knows precisely which view to render, ensuring that the relevant information is presented consistently across various platforms:
+
+### Display
+
+This view provides basic information about the NFT suitable for listing or display purposes. When the `Display` type is invoked, it dynamically assembles the visual and descriptive information that is typically needed for showcasing the NFT in marketplaces or collections.
+
+```cadence
+case Type<MetadataViews.Display>():
+    return MetadataViews.Display(
+        name: self.name,
+        description: self.description,
+        thumbnail: MetadataViews.HTTPFile(
+            url: self.thumbnail
+        )
+    )
+```
+
+### Editions
+
+The `Editions` view provides intricate details regarding the particular release of an NFT. This can include information about the number of copies in an edition, the sequence number of the specific NFT within that edition, or if it is part of a limited series. When the `Editions` view is queried, it retrieves this data, essential for collectors to understand the rarity and exclusivity of the NFT they are interested in.
```

Comment:
```suggestion
The `Editions` view provides intricate details regarding the particular release of an NFT. This can include information about the number of copies in an edition, the specific NFT's sequence number within that edition, or its inclusion in a limited series. When the `Editions` view is queried, it retrieves this data, providing collectors with the information they need to comprehend the rarity and exclusivity of the NFT they are interested in.
```